src/strategies_and_params/replicastrategy.jl

```diff
@@ -63,27 +64,40 @@ Column names in the report are of the form `c{i}_dot_c{j}` for vector-vector ove
 See [`lomc!`](@ref), [`ReplicaStrategy`](@ref) and [`AbstractHamiltonian`](@ref) (for an
 interface for implementing operators).
 
-If `transform` is not `nothing` then this strategy calculates the overlaps with respect to a 
-similarity transformation `G` of the Hamiltonian (see e.g. [`GutzwillerSampling`](@ref).)
+# Transformed Hamiltonians
+
+If a transformed Hamiltonian `G` has been passed to [`lomc!`](@ref) then overlaps can be calculated by
+passing the same transformed Hamiltonian to `AllOverlaps` by setting `transform=G`. A warning is given 
+if these two Hamiltonians do not match.
+
+Implemented transformations are:
+
+    * [`GutzwillerSampling`](@ref)
+    * [`GuidingVectorSampling`](@ref)
```

Comment:
This does not render well, see html preview. I think it is just the indentation that is off.
```suggestion
 * [`GutzwillerSampling`](@ref)
 * [`GuidingVectorSampling`](@ref)
```